automation_tools/__init__.py

```diff
@@ -2088,36 +2083,22 @@ def product_install(distribution, certificate_url=None, selinux_mode=None, sat_v
             distribution, ', '.join(distributions)))
         sys.exit(1)
 
-    if (
-        distribution == 'satellite6-cdn' and
-        sat_version not in ('6.3', '6.4', '6.5', '6.6', '6.7')
-    ):
-        raise ValueError(
-            "Satellite version should be in [6.3, 6.4, 6.5, 6.6, 6.7]"
-        )
-
-    if selinux_mode is None:
-        selinux_mode = os.environ.get('SELINUX_MODE', 'enforcing')
-
-    if distribution == 'satellite6-iso':
-        iso_url = os.environ.get('ISO_URL')
-        if iso_url is None:
-            print('The ISO_URL environment variable should be defined')
-            sys.exit(1)
+    if distribution == 'cdn' and sat_version not in ('6.3', '6.4', '6.5', '6.6', '6.7'):
+        raise ValueError("Satellite version should be in [6.3, 6.4, 6.5, 6.6, 6.7]")
```

Comment:
does 6.3 need to be removed from this section?

Reply:
it does need to be, but I am removing 6.3 functionality so it won't work properly anyway

Reply:
it doesn't need to be, I am removing 6.3 functionality in another PR https://github.com/SatelliteQE/automation-tools/pull/843 (merged)

So I better rebase to what's been merged

Reply:
Rebased. (6.3 gone)